community/GOVERNANCE.md

```diff
@@ -141,36 +141,40 @@ The specifications produced will adhere to the following:
   apply and the "major" version number will change. The net effect of this is
   that the "minor" version number will always be zero and the `specversion`
   string will always be of the form `X.0`.
-- Additionally, the working group has decided that "patch" changes that are
-  trivial in nature (e.g. typos) can be merged directly into the latest
-  release without a version number change and without creating a new formal
-  "patch" branch/release. Non-trivial "patch" changes will necessitate the
-  creation of a new "patch" branch (e.g. `1.0.2-wip`) and release cycle that
-  will eventually result in an official "patch" release (e.g. `1.0.2`)
 
 Note that these rules do not apply to the
 [documented extensions](../cloudevents/documented-extensions.md).
 
+All versions are tagged from the `main` branch, but the tag only applies to
+the "subject" of the release - the directory containing the information
+covered by that release (e.g. `discovery` or `cloudevents`). The
+[CloudEvents web site](https://cloudevents.io/) takes appropriate content from
+each tagged version. (If the directory containing the information covered
```

Comment:
I'm having trouble parsing the sentence in parens. Did you leave off a "is in a nested sub-directory"  after "by the release" ?

Reply:
Whoops, yes. Fixing.